setting-up-bacalhau-cluster/docker-compose/multi-region/docker-compose.yml

```diff
@@ -0,0 +1,158 @@
+name: bacalhau-multi-region
+
+x-common-env-variables: &common-env-variables
+  MINIO_ROOT_USER: "minioadmin"
+  MINIO_ROOT_PASSWORD: "minioadmin"
+  AWS_ACCESS_KEY_ID: "minioadmin"
+  AWS_SECRET_ACCESS_KEY: "minioadmin"
+  BACALHAU_DISABLEANALYTICS: true
+
+services:
+  orchestrator:
+    image: ghcr.io/bacalhau-project/bacalhau:latest
+    hostname: orchestrator
+    command: serve -c /etc/bacalhau/config.yaml --name orchestrator
+    environment: *common-env-variables
+    ports:
+      - "8438:8438"
+      - "1234:1234"
+      - "4222:4222"
+    networks:
+      - bacalhau-network
+    volumes:
+      - ./config/orchestrator.yaml:/etc/bacalhau/config.yaml
+    healthcheck:
+      test: ["CMD", "bacalhau", "agent", "alive"]
+      interval: 2s
+      timeout: 2s
+      retries: 3
+      start_period: 5s
```

Comment:
One minor suggestion: I found that increasing the number of retries is better for environments with limited resources.  I usually put it to at least 10.